packages/patternfly-4/react-core/src/components/AboutModal/AboutModal.tsx

```diff
@@ -76,34 +79,45 @@ export class AboutModal extends React.Component<AboutModalProps, ModalState> {
     }
   };
 
+  getElement = (appendTo: HTMLElement | (() => HTMLElement)) => {
+    if (typeof appendTo === 'function') {
+      return appendTo();
+    }
+    return appendTo;
+  };
+
   componentDidMount() {
     const container = document.createElement('div');
+    const target: HTMLElement = this.getElement(this.props.appendTo);
     this.setState({ container });
-    document.body.appendChild(container);
-    document.addEventListener('keydown', this.handleEscKeyClick, false);
+    target.appendChild(container);
+    target.addEventListener('keydown', this.handleEscKeyClick, false);
```

Comment:
Considering the default value for `appendTo` is null, it appears that `target.appendChild` could generate a null exception.

Reply:
I can add a check before calling these making sure it's not null.
But when target is `null` it means `window.document` is not defined which is odd.

Reply:
I saw the check for `document !== 'undefined'`, so assumed there's a condition where it could be null. Maybe just use `appendTo: document.body`?